### PR TITLE
Add MySQL server to resources evaluated by AzCloud

### PR DESCRIPTION
**Pull request by mitprasoon**
The MySQL servers associated with a subscription are evaluated and
processed. A record is generated, with the value of `record_type` set to
`mysql_server` in the `ext` bucket and `record_type` set to `rdbms` set
in the `com` bucket of the record.

### CODE REVIEW

File: cloudmarker/clouds/azcloud.py [hunk 1]
```diff
@@ -111,6 +114,9 @@ def read(self):
 
                     _get_record(resource_list, 'resource',
                                 subscription_id, self._max_recs),
+
+                    _get_record(mysql_server_list, 'my_sql_server',
+                                subscription_id, self._max_recs),
```
Comment (susam) [inline cloudmarker/clouds/azcloud.py]: Could you please rebase this pull request on the latest `master` branch? This part of the change seems to be based on an older `master` branch. A few merge conflicts have to be resolved after rebasing this on the latest `master`.

[the remark this answers is not in the document]
Comment (susam) [reply cloudmarker/clouds/azcloud.py]: Changing the record type to `mysql_server` will require an update to the commit message as well.

File: pkg-requirements.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ asn1crypto==0.24.0
 azure-common==1.1.20
 azure-mgmt-compute==5.0.0
 azure-mgmt-network==2.7.0
+azure.mgmt.rdbms==1.8.0
```
[the remark this answers is not in the document]
Comment (mitprasoon) [reply pkg-requirements.txt]: Added `azure-mgmt-rdbms` to `usr-requirements.txt`

File: cloudmarker/clouds/azcloud.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mitprasoon) [reply cloudmarker/clouds/azcloud.py]: Changed the record type to `mysql_server `
Also, updated the commit message

Comment (mitprasoon) [reply cloudmarker/clouds/azcloud.py]: Rebased the PR on latest `master` branch

**Pull request merged**